idalib/src/idb.rs

```diff
@@ -379,6 +381,29 @@ impl IDB {
         }
     }
 
+    pub fn strings(&self) -> StringList {
+        StringList::new(self)
+    }
+
+    pub fn get_string(&self, index: usize) -> Option<String> {
```

Comment:
I'd move this to `StringList` as `get_by_index`

Reply:
Sure, the only problem I have is that I need to access the `get_bytes()` method from within `StringList`. Is there a clean way to achieve this with the current architecture that uses `PhantomData`?

Reply:
I guess we have two options:
- Reuse the implementation of `get_bytes` inline inside this function. If we find this is a repeated pattern later, we can figure out how to reuse it.
- Implement the logic in C++ such that we return a String for a given index.

Reply:
Done! I went for the first option.